docs/bash.md

```diff
@@ -887,6 +1079,160 @@ const {pid} = $`npm run build`;
 
 [More info.](termination.md#inter-process-termination)
 
+### CLI arguments
+
+```js
+// zx
+const {myCliFlag} = argv;
+```
+
+```js
+// Execa
+import minimist from 'minimist';
+
+const {myCliFlag} = minimist(process.argv.slice(2));
+```
+
+[More info.](https://github.com/minimistjs/minimist)
+
+### CLI prompts
+
+```sh
+# Bash
+read -p "Question? " answer
+```
+
+```js
+// zx
+const answer = await question('Question? ');
+```
+
+```js
+// Execa
+import input from '@inquirer/input';
+
+const answer = await input({message: 'Question?'});
+```
+
+[More info.](https://github.com/SBoudrias/Inquirer.js)
+
+### CLI spinner
+
+```sh
+# Bash does not provide with a builtin spinner
+```
+
+```js
+// zx
+await spinner(() => $`node script.js`);
+```
+
+```js
+// Execa
+import {oraPromise} from 'ora';
+
+await oraPromise($`node script.js`);
+```
+
+[More info.](https://github.com/sindresorhus/ora)
+
+### Sleep
+
+```sh
+# Bash
+sleep 5
+```
+
+```js
+// zx
+await sleep(5000);
+```
+
+```js
+// Execa
+import {setTimeout} from 'node:timers/promises';
+
+await setTimeout(5000);
+```
+
+[More info.](https://nodejs.org/api/timers.html#timerspromisessettimeoutdelay-value-options)
+
+### Globbing
+
+```sh
+# Bash
+ls packages/*
+```
+
+```js
+// zx
+const files = await glob(['packages/*']);
+```
+
+```js
+// Execa
+import {globby} from 'globby';
```

Comment:
https://nodejs.org/api/fs.html#fspromisesglobpattern-options

Reply:
Fixed in 0cf80f0a